app/javascript/dashboard/components/ui/HelperTextPopup.vue

```diff
@@ -0,0 +1,24 @@
+<script setup>
+defineProps({
+  message: {
+    type: String,
+    default: '',
+  },
+});
+</script>
+<template>
+  <div class="relative group w-[inherit] whitespace-normal z-20">
+    <fluent-icon
+      icon="info"
+      size="14"
+      class="mt-0.5 text-slate-600 absolute group dark:text-slate-400"
```

Comment:
```suggestion
      class="mt-0.5 text-slate-600 absolute dark:text-slate-400"
```
Does this need the group class?

Reply:
Yes, not needed. Added it mistakenly.